spring-core/src/main/java/org/togglz/spring/activation/SpringProfileActivationStrategy.java

```diff
@@ -0,0 +1,64 @@
+package org.togglz.spring.activation;
+
+import java.util.Arrays;
+import java.util.List;
+import org.springframework.context.ApplicationContext;
+import org.togglz.core.activation.Parameter;
+import org.togglz.core.activation.ParameterBuilder;
+import org.togglz.core.repository.FeatureState;
+import org.togglz.core.spi.ActivationStrategy;
+import org.togglz.core.user.FeatureUser;
+import org.togglz.core.util.Strings;
+import org.togglz.spring.util.ContextClassLoaderApplicationContextHolder;
+
+/**
+ * <p>
+ * An activation strategy based on the profiles that are active within the Spring environment.
+ * </p>
+ *
+ * @author Alasdair Mercer
+ */
+public class SpringProfileActivationStrategy implements ActivationStrategy {
+
+    public static final String ID = "spring-profile";
+    static final String NAME = "Spring Profile";
```

Comment:
I suggest to either make this private or to inline the field. Letting getName() return a literal is fine for me. ;-)